.github/workflows/awsfulltest.yml

```diff
@@ -0,0 +1,36 @@
+name: nf-core AWS full size tests
+# This workflow is triggered on releases.
+# It runs the -profile 'test_full' on AWS batch
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  run-awstest:
+    if: github.repository == 'nf-core/atacseq'
+    name: Run AWS test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Miniconda
+        uses: goanpeca/setup-miniconda@v1.0.2
+        with:
+          auto-update-conda: true
+          python-version: 3.7
+      - name: Install awscli
+        run: conda install -c conda-forge awscli
+      - name: Start AWS batch job
+        env:
+          AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
+          AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+          TOWER_ACCESS_TOKEN: ${{secrets.AWS_TOWER_TOKEN}}
+          #AWS_JOB_DEFINITION: ${{secrets.AWS_JOB_DEFINITION}}
+          AWS_JOB_QUEUE: ${{secrets.AWS_JOB_QUEUE}}
+          AWS_S3_BUCKET: ${{secrets.AWS_S3_BUCKET}}
+        run: | # Submits job to AWS batch using a 'nextflow-4GiB' job definition. Setting JVM options to "-XX:+UseG1GC" for more efficient garbage collection when staging remote files.
+          aws batch submit-job \
+          --region eu-west-1 \
+          --job-name nf-core-atacseq \
+          --job-queue $AWS_JOB_QUEUE \
+          --job-definition nextflow-4GiB \

```

Comment:
We may not need this larger instance and the garbage collection (`-XX:+UseG1GC`) after we have implemented Paolo's config trick.

Reply:
Hi yes, that is right, but this is not yet implemented in the `awsbatch.config`. That's the same reason why the env variable $AWS_JOB_DEFINITION is commented out above.

Reply:
Ah ok. Thanks 👍